frontend/src/components/categorycard/categorycard.module.css

```diff
@@ -0,0 +1,37 @@
+.categorycard_container{
+    margin: 1rem;
+    width: 90%;
+    position: relative;
+    box-shadow: 0 5px 20px #474747;
+    transition: transform 200ms, opacity 400ms;
+    opacity: 0.7;
+}
+
+.categorycard_container:hover{
+    transform: scale(1.2);
```

Comment:
![image](https://user-images.githubusercontent.com/44545218/115960016-bc3a1800-a52c-11eb-8f4f-2485d2599d94.png)
Sometimes there is overlap with other playlist, look into the property

Reply:
done